bsm_helper/src/bsm_helper.cpp

```diff
@@ -0,0 +1,40 @@
+#include "bsm_helper/bsm_helper.h"
+
+namespace BSMHelper{
+
+
+    std::string BSMHelper::bsmIDtoString(std::vector<uint8_t, std::allocator<uint8_t>> id)
+    {
+
+        std::string id_test;
+
+        for(size_t i =0; i < id.size();i++)
+        {
+            id_test.append(std::to_string(id.at(i)));
+        }
+
+        std::stringstream int_id(id_test);
+
+        unsigned long id_num;
+
+        int_id >> id_num;
```

Comment:
Converting to a string and then pushing into a long just to convert back to a string seems pretty round about. Couldn't you just push each byte directly into the "unsigned long" using bit wise operators?